### PR TITLE
Improve debug log naming

### LINKED ISSUE #2690: Improve debug log naming
## Abstract

Raiden automatically writes a debug log file on each run. Currently this file is called `raiden-debug.log` with an optional numeric suffix for log rotation.

## Motivation

Having this simplistic name becomes problematic when running multiple nodes at once or multiple times in quick succession.

Therefore it would be helpful to include more details in the filename to avoid conflicts and make them easier to identify.

## Specification

Change the debug file name to follow this template (slightly pseudo code):
`f'raiden-debug-node_{pex(node_addr)}-net_{net_id}-{datetime.now().isoformat()}.log'`

## Backwards Compatibility

This is a debug only feature, so backwards compatibility is not relevant.
We will need to update the issue template text asking users to submit those files slightly though.

### PR DESCRIPTION
**Pull request by dergoegge**
Closes #2690.

### CODE REVIEW

**Comment by rakanalh** [comment]
Reviewing this.

**Comment by dergoegge** [comment]
@rakanalh I went for a totally different approach so i rebased and deleted the commits you reviewed. Instead of renaming the logfile configure_logging uses the database path. There is further discussion about this in the issue #2690.

File: raiden/ui/app.py [hunk 1]
```diff
@@ -336,14 +337,23 @@ def run_app(
...
-    config['database_path'] = database_path
+
+    config['database_path'] = os.path.join(database_path, f'v{RAIDEN_DB_VERSION}_log.db')
+
+    log_file_path = os.path.join(database_path, 'debug.log')
+    log_file_path = kwargs.get('log_file') if kwargs.get('log_file') else log_file_path
+    configure_logging(
+        kwargs.get('log_config'),
+        log_json=kwargs.get('log_json'),
+        log_file=log_file_path,
```
Comment (ulope) [inline raiden/ui/app.py]: The correct parameter is `debug_log_file_name`.

File: raiden/ui/app.py [hunk 2]
```diff
@@ -341,17 +341,19 @@ def run_app(
 
     config['database_path'] = os.path.join(database_path, f'v{RAIDEN_DB_VERSION}_log.db')
 
+    log_file_path = os.path.join(database_path, 'debug.log')
+    log_file_path = kwargs.get('log_file') if kwargs.get('log_file') else log_file_path
```
Comment (rakanalh) [inline raiden/ui/app.py]: This line could become:
`log_file_path = kwargs.get('log_file', log_file_path)`

File: raiden/ui/app.py (hunk 1 above)
Comment (dergoegge) [reply raiden/ui/app.py]: should the ```log_file``` param still be assigned something or just ```debug_log_file_name=log_file_path ```?

**Comment by dergoegge** [comment]
@ulope  how do i get ```token_network_registry``` after the account prompt because that is being assigned way later?

**Comment by hackaugusto** [comment]
@dergoegge `token_network_registry.address` is the address of the smart contract, defined in the constructor call of `token_network_registry`.

This value is either passed as an argument or the the default is used, which you can get with the expression `registry_contract_address or contract_addresses[CONTRACT_TOKEN_NETWORK_REGISTRY]`.

**Comment by dergoegge** [comment]
@hackaugusto but ```contract_addresses``` is also assigned later or am i missing your point?

**Comment by hackaugusto** [comment]
> contract_addresses is also assigned later or am i missing your point?

Yes, yes, but it's should be easy to move it up too.

**Comment by dergoegge** [comment]
I dont understand why the ci fails. ```kwargs.get('log_config')``` in app.py should result in the same value (a dict) as previously ```self._options['log_config']```in runners.py for the ```logger_level_config```.

**Comment by LefterisJP** [comment]
I assume this PR is abandoned. @dergoegge will you start this again or rebase in master and handle the comments by @rakanalh and @ulope or should we close this?

**Comment by dergoegge** [comment]
@LefterisJP i will try. I dont have much time at the moment. If i dont get it done within one week you can close it.

**Comment by dergoegge** [comment]
@LefterisJP The unit tests are failing because i switched the log calls in the `AccountManager` to `click.secho` calls (as suggested [here](https://github.com/raiden-network/raiden/issues/2690#issuecomment-428152513)

> No, that should be fine. We only need to switch from using logging to prints (or rather `click.secho` calls) in the `AccountManager`. Shouldn't be a big issue since it only has two logging calls IIRC.

)

I dont really want to mess your tests up... what do i do here?

**Comment by hackaugusto** [comment]
@dergoegge thank you very much for the contribution. I'm sorry, but I'm closing the PR because the base branch is too outdated.

**Pull request closed without merging**